react/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+ "compilerOptions": {
+   "jsx": "react",
+   "esModuleInterop": true
+ } 
+}
```

Comment:
Change to use the default config

https://github.com/vtex-apps/react-app-template/blob/master/react/tsconfig.json